pom.xml

```diff
@@ -937,7 +937,7 @@
               <systemId>http://maven.apache.org/xsd/decoration-1.0.0.xsd</systemId>
             </validationSet>
             <validationSet>
-              <dir>src/main/resources</dir>
+              <dir>src/test/resources</dir>
```

Comment:
We need to keep this file and do brute force as last resort of class resolving till checkstyle8 .

#3660 .